llvm/docs/GlobalISel/GenericOpcode.rst

```diff
@@ -690,6 +690,10 @@ G_SPLAT_VECTOR
 
 Create a vector where all elements are the scalar from the source operand.
 
+The type of the operand must be equal to or smaller than the vector element
+type. If the operand is smaller than the vector element type, the scalar is
+implicitly truncated to the vector element type.
```

Comment:
extended?

Reply:
I got it all backwards 😆 . The operand should be allowed to be greater than the size of the vector element.